abuseipdb/plugin.spec.yaml

```
@@ -12,13 +12,12 @@ resources:
   source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/abuseipdb
   license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
   vendor_url: https://www.abuseipdb.com
-  documentation_url: https://docs.abuseipdb.com
 tags:
 - ip
 - intelligence
 - abuse
 hub_tags:
-  use_cases: [report_ip, get_blacklisted_ip]
+  use_cases: [investigations, reporting_and_analytics]
```

Make this `alert_enrichment, threat_intel`
```suggestion
  use_cases: [alert_enrichment, threat_intel]
```